clarify.toml

```diff
@@ -1,3 +1,10 @@
+[spdx]
```

Comment:
This is to fix the failure when running `make iamges`. The `license-scan` was not happy with the Pixar license.

```
1.384 Error: detected license "Pixar" from LICENSE-APACHE is not present in the license field "MIT OR Apache-2.0" for allocator-api2
```
Referring to [this code](https://github.com/bottlerocket-os/bottlerocket-core-kit/pull/331/files#diff-ae237b2dd44b0855242f2349cfe51b15b570d4ba0470b0b84bd24e5f3820d07a) in core-kit for the fix.